webapp/admin/maintenance.py

```
@@ -626,7 +630,90 @@ def optimize_database():
+                    line = line.strip()
+                    if line and not line.startswith("#") and "=" in line:
+                        key, _, value = line.partition("=")
+                        env_vars[key.strip()] = value.strip()
+
+                return jsonify({
+                    "content": content,
+                    "env_vars": env_vars,
+                    "path": str(env_file_path),
+                })
+
+            except Exception as exc:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)